webui/src/pages/extension-detail/extension-detail.tsx

```diff
@@ -211,64 +215,66 @@ export class ExtensionDetailComponent extends React.Component<ExtensionDetailCom
         </React.Fragment>;
     }
 
-    protected renderHeader(extension: Extension): React.ReactNode {
-        const themeClass = extension.galleryTheme === 'dark' ? this.props.classes.darkTheme : this.props.classes.lightTheme;
-        return <Box overflow='auto'>
-            <Typography variant='h5' className={this.props.classes.titleRow}>
-                {extension.displayName || extension.name} {extension.preview ?
-                    <span className={`${this.props.classes.preview} ${themeClass}`}>preview</span>
-                    : ''}
-            </Typography>
-            <Box display='flex' className={`${themeClass} ${this.props.classes.info}`}>
-                <Box className={this.props.classes.alignVertically} >
-                    {this.renderAccessInfo(extension, themeClass)}&nbsp;<RouteLink
-                        to={addQuery(ExtensionListRoutes.MAIN, [{ key: 'search', value: extension.namespace }])}
-                        className={`${this.props.classes.link} ${themeClass}`}
-                        title={`Namespace: ${extension.namespace}`} >
-                        {extension.namespace}
-                    </RouteLink>
+    protected renderHeader(extension: Extension, shouldBeLight: boolean): React.ReactNode {
```

Comment:
`shouldBeLight` seems a bit misleading, I would rename it to `darkTheme`, or use a parameter `themeType` of type `'light' | 'dark'` that you can pass to the `TextDivider` components.